src/main/java/com/powsybl/openloadflow/sensi/AcSensitivityAnalysis.java

```diff
@@ -180,6 +180,12 @@ public void analyse(Network network, List<PropagatedContingency> contingencies,
                 .collect(Collectors.toSet());
 
         // create AC engine
+        boolean hasTransformerBusTargetVoltage = hasTransformerBusTargetVoltage(factorHolder, network);
+        if (hasTransformerBusTargetVoltage) {
+            // if we have at least one bus target voltage linked to a ratio tap changer, we activate the transformer
```

Comment:
So you think that the user should give a bus target voltage related to a transformer AND explicitly says that transformer control is enabled? 
What I don't get in your point is: what is for you the starting point to compute sensitivity values? In AC, it is quite important. In the code, the starting point is after transformer rounding. You prefer having both modes: after the rounding if we have the control enabled in the parameters or just from the initial network?

Reply:
No no I think it's perfectly OK to compute sensitivities after the rounding.

What I mean is that: 
1. if you run a simple loadflow with voltage control disabled, you will get voltage values without tap changes, OK
2. if you run a sensitivity analysis with the same parameters (=voltage control disabled), now you will get different values for voltages (and sensitivities around those different values), because taps will have moved

I think it's weird, if you run the computation with the same parameters, I would expect to obtain the same voltage values.
If voltage control is disabled, I think I would expect to have 0 as sensitivity value (if we do this, maybe we could add a WARN log ?).

Reply:
Yes I can do that.

Reply:
Second thought: for sensitivity calculation linked to a phase shifter, we don’t use the loadflow parameters. We can compute the sensitivity even if the regulation is not active. So it is still coherent to rely on the loadflow parameter on our use case, I think yes but what is our opinion?